docs/reference/indices/put-component-template.asciidoc

```diff
@@ -85,7 +85,29 @@ except before the opening curly bracket.
 `<component-template>`::
 (Required, string)
 Name of the component template to create.
-
++
+[IMPORTANT]
+====
+{es} includes the following built-in component templates:
+
+// tag::built-in-component-templates[]
+- `logs-mappings`
+- `logs-settings`
+- `metrics-mappings`
+- `metrics-settings`
+- `synthetics-mapping`
+- `synthetics-settings`
+// end::built-in-component-templates[]
+
+The {fleet-guide}/fleet-overview.html[{agent}] uses these templates to configure
+backing indices for its data streams. If you use the {agent} and want to
+overwrite one of these templates, set the `version` for your replacement
+template to `1` or greater.
```

Comment:
I would recommend that we just say "higher than the current version", so that if/when we bump the internal version we don't forget to bump it here and cause confusion, what do you think?

Reply:
Good thinking. I'll amend it to `higher than the current version`.